README.md

```diff
@@ -1 +1,55 @@
-# Recipe-Nutri
\ No newline at end of file
+# Recipe-Nutri
+
+## Table of Contents
+-Description
+-Features
+-Usage
+-Technologies
+-Screenshot
+-Link
+
+## Description
+RecipeNutri is a web application that allows users to find and explore recipes while providing detailed nutritional information. Users can search for recipes by recipe name, view cooking instructions, and get accurate nutrition facts for their chosen dishes. This README provides an overview of the application, instructions for usage, and details for developers.
```

Comment:
In the description, we need to include the user story, but from our point of view to include our motivations and what we learned. I use the acceptance criteria to help me write this part. 

Project Requirements
You and your group will use everything you’ve learned over the past six weeks to create a real-world front-end application that you’ll be able to showcase to potential employers. The user story and acceptance criteria will depend on the project that you create, but your project must fulfil the following requirements:

Use a CSS framework other than Bootstrap.

Be deployed to GitHub Pages.

Be interactive (i.e: accept and respond to user input).

Use at least two server-side APIs.

Does not use alerts, confirms, or prompts (use modals).

Use client-side storage to store persistent data.

Be responsive.

Have a polished UI.

Have a clean repository that meets quality coding standards (file structure, naming conventions, follows best practices for class/id-naming conventions, indentation, quality comments, etc.).

Have a quality README (with unique name, description, technologies used, screenshot, and link to deployed application).

Finally, You must add your project to the portfolio that you created in Module 2. ***WE NEED TO ALL DO THIS - WHOOPS!***